internal/handlers/pg/msg_dropdatabase.go

```diff
@@ -41,7 +43,15 @@ func (h *Handler) MsgDropDatabase(ctx context.Context, msg *wire.OpMsg) (*wire.O
 	}
 
 	res := must.NotFail(types.NewDocument())
-	err = h.pgPool.DropDatabase(ctx, db)
+
+	err = h.pgPool.InTransaction(ctx, func(tx pgx.Tx) error {
+		if err := pgdb.DropDatabase(ctx, tx, db); err != nil {
+			return err
+		}
+
+		return nil
```

Comment:
```suggestion
		return pgdb.DropDatabase(ctx, tx, db)
```

To not make it more complicated by shadowing `err`